Physics/Dynamics/ProductionFormFactor.cpp

```diff
@@ -0,0 +1,237 @@
+// Copyright (c) 2013, 2017 The ComPWA Team.
+// This file is part of the ComPWA framework, check
+// https://github.com/ComPWA/ComPWA/license.txt for details.
+
+#include <cmath>
+#include <iterator>
+#include <numeric>
+#include <vector>
+
+#include "Coupling.hpp"
+#include "Physics/HelicityFormalism/HelicityKinematics.hpp"
+#include "ProductionFormFactor.hpp"
+
+namespace ComPWA {
+namespace Physics {
+namespace Dynamics {
+
+ProductionFormFactor::ProductionFormFactor(std::string name,
+                                           std::string daug1Name,
+                                           std::string daug2Name,
+                                           std::shared_ptr<ComPWA::FitParameter> mass1,
+                                           std::shared_ptr<ComPWA::FitParameter> mass2,
+                                           std::shared_ptr<ComPWA::FitParameter> radius,
+                                           ComPWA::Spin orbitL,
+                                           FormFactorType ffType)
+    : AbstractDynamicalFunction(name) {
+
+  LOG(TRACE) << "ProductionFormFactor::Factory() | Construction of " << name
+             << ".";
+
+  SetDaughter1MassParameter(mass1);
```

Comment:
initialize the members via `ProductionFormFactor::ProductionFormFactor(...) : ..... {}`